src/Gemfile

```diff
@@ -1,5 +1,6 @@
 # frozen_string_literal: true
 source "https://rubygems.org"
 
+gem "puma"
```

Comment:
Because the sinatra app is sorta kinda packaged like a gem, maybe declare this in `dcob.gemspec` where the other gem dependencies are declared?